default.nix

```diff
@@ -46,6 +46,7 @@ in {
       cardano-wallet-core-integration
       cardano-wallet-http-bridge
       cardano-wallet-jormungandr
+      cardano-wallet-shelley
```

Comment:
why it is needed?

Reply:
Lets me use ghci. I can move it to another PR.